javascript/javascript2/week1/homework app2/main.js

```diff
@@ -0,0 +1,27 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+let ul = document.querySelector(".listOfProducts");
+
+function renderProducts(ourProducts) {
+    for (let i = 0; i < ourProducts.length; i++) {
+        const productNames = ourProducts[i];
```

Comment:
just a minor note: I would favor using singular names for single values and in this case is also not really a productName you are getting but  a "product" itself 🐱‍👓 if someone(also future you) was reading and using your code afterwards it wouldn't be that clear 😇

Reply:
Thank you kindly for your review. Your comments make sense and i fixed all what you said.  I'll pay attention more for naming elements next time!